examples/basic-tracer-node/index.js

```diff
@@ -17,14 +17,14 @@ if (EXPORTER.toLowerCase().startsWith('z')) {
   exporter = new JaegerExporter(options);
 }
 
-const tracer = new BasicTracer();
+// Initialize the OpenTelemetry APIs to use the BasicTracer bindings
+opentelemetry.initGlobalTracerFactory(new BasicTracerFactory());
```

Comment:
Should this use `BasicTracerFactory.instance()` instead of directly calling `new`?

Reply:
IMO this (initialization of global tracer factory and `BasicTracerFactory`) seems a little burdensome for the end users. I was thinking something like this:
```js
// In case of BasicTracer
const opentelemetry = require('@opentelemetry/tracing');
opentelemetry.getTracerFactory().getTracer(); // should return the default BasicTracer.
opentelemetry.getTracerFactory().getTracer('mongodb'); // with name
opentelemetry.getTracerFactory().getTracer('redis', '1.0.0'); // with name and version
opentelemetry.getTracerFactory().addSpanProcessor(...);

// In case of NodeTracer
const opentelemetry = require('@opentelemetry/node');
opentelemetry.getTracerFactory().getTracer(); // should return the default NodeTracer.

// In case of WebTracer
const opentelemetry = require('@opentelemetry/web');
opentelemetry.getTracerFactory().getTracer(); // should return the default WebTracer.
```

The `getTracerFactory` function is responsible for returning the corresponding `TracerFactory` and we should create (and assign) the `TracerFactory` instance statically once you load the library (maybe in `index.ts`). WDYT?